src/libs/Firebase/index.native.ts

```diff
@@ -23,6 +23,11 @@ const startTrace: StartTrace = (customEventName) => {
         .startTrace(customEventName)
         .then((trace) => {
             trace.putAttribute('accountID', session?.accountID?.toString() ?? 'N/A');
+            if (attributes) {
```

Comment:
we already obtain session properties here, what stops us from calling this `getProperties()` here directly?

Reply:
It was the initial idea, but in `getProperties`, when we call for getting `allReports` from `ReportConnection`, there were some cases, (e.g. in the EmojiTrie ) where we ran `Timing.start` before initializing the component. This caused both unit and performance tests to fail. So we need solution to control which methods we want to call in Timing function. 

That's why we decided to pass attributes explicitly to `Timing.start`. This approach also gives us more flexibility when we want to collect additional data for specific metrics.

Reply:
Hm but these are not event specific, they're the same as an `accountId` as they will be required for all the events.

Have we already tried fixing tests in the first place?

Reply:
I changed the approach, I moved logic directly to firebase file and mocked the Timing function in the tests, as it was initially planned. I run the reassure tests and the result where not impacted, so we should be fine :)

Thanks for the input!